src/web/command_control/client/components/MissionSettings.tsx

```diff
@@ -20,9 +20,10 @@ export interface MissionParams {
 	missionType: 'editing' | 'polygon-grid' | 'lines' | 'exclusions'
 	numBots: number,
 	numGoals: number,
-	spacing: number,
-	orientation: number,
+    pointSpacing: number,
```

Comment:
Spacing looks off here

Reply:
I just re-indented the lines. Here's how they look in VS Code: 
![image](https://github.com/jaiarobotics/jaiabot/assets/109538793/602ad54b-fe38-46ed-9e74-e7e883c7a49b)